README.md

```diff
@@ -1,21 +1,21 @@
 # NixOS module for NVIDIA Jetson devices
 
-This repository packages components from NVIDIA's [Jetpack SDK](https://developer.nvidia.com/embedded/jetpack) for use with NixOS, including:
- * Platform firmware flashing scripts
- * A 5.10 Linux kernel from NVIDIA, which includes some open-source drivers like nvgpu
- * An [EDK2-based UEFI firmware](https://github.com/NVIDIA/edk2-nvidia)
- * ARM Trusted Firmware / OP-TEE
- * Additional packages for:
-   - GPU computing: CUDA, CuDNN, TensorRT
-   - Multimedia: hardware accelerated encoding/decoding with V4L2 and gstreamer plugins
-   - Graphics: Wayland, GBM, EGL, Vulkan
-   - Power/fan control: nvpmodel, nvfancontrol
+This repository packages components from NVIDIA's [ Jetpack SDK ] (https://developer.nvidia.com/embedded/jetpack) for use with NixOS, including:
```

Comment:
Can you remove these formatting changes?

Reply:
Fixed in latest